examples/README.md

```diff
@@ -4,7 +4,7 @@
 
 ## Web scraping
 - [Puppetron](https://github.com/cheeaun/puppetron) - Demo site that shows how to use Puppeteer and Headless Chrome to render pages. Inspired by [GoogleChrome/rendertron](https://github.com/GoogleChrome/rendertron).
- - [Thal](medium.com/@e_mad_ehsan/getting-started-with-puppeteer-and-chrome-headless-for-web-scrapping-6bf5979dee3e) - Getting started with Puppeteer and Chrome Headless for Web Scraping.
```

Comment:
It seems this link just luck the protocol part(`https://`), this works:

https://medium.com/@e_mad_ehsan/getting-started-with-puppeteer-and-chrome-headless-for-web-scrapping-6bf5979dee3e

Reply:
ok i got it.